File: mario_Comfortable.py
height = input("Enter height of pyramid (integer from 0-23):")

#loop to ensure accepted user input is a non-negative integer from 0-23
while height.isnumeric()==False or int(height) < 0 or int(height) > 23:
    height = input("Enter height of pyramid (integer from 0-23):")

#loop printing a combination of spaces and hashes per row (or iteration)
for i in range(1, int(height)+1):
    print((int(height)-i)*" ",i*"#","  ",i*"#")

File: marioComfortable.c
#include <stdio.h>

int main(void)
{
  int height;

  do
  {
      printf("Enter height of pyramid (integer from 0-23):");
      scanf("%d", &height);
      fflush(stdin); //prevents infinite loop when string is entered
  }
  while (height < 0 || height > 23); //accepts int value from 0-23

  for (int i = 0; i < height; i++)
    {
        for (int space = height - i; space > 1; space--) //loop prints spaces to align pyramids with right edge of window
        {
            printf(" ");
        }
        for (int hash = 0; hash < i + 1; hash++) //loop prints hashes for 1st pyramid half
        {
            printf("#");
        }
        printf("  "); //double space between pyramid halves
	for (int hash = 0; hash < i + 1; hash++) //loop prints hashes for 2nd pyramid half
        {
            printf("#");
        }
        printf("\n");
    }
}
